lib/notification.rb

```diff
@@ -0,0 +1,24 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'dotenv'
+
+class Notification
+
+  def initialize
+    account_sid = "AC0ca58c2cacff589c5395eaef6ba4bf53"
+    auth_token = "b1e17879c8440d0697930ba8aac116b3"
+    @client = Twilio::REST::Client.new ENV['ACCOUNT_SID'], ENV['AUTH_TOKEN']
+  end
+
+  def send_sms(message)
+    client.account.messages.create(
+      :from => ENV['TWILIO_PHONE'],
+      :to => ENV['TWILIO_DESTINATION_PHONE'],
+      :body => message
+      )
```

Comment:
Style/ClosingParenthesisIndentation: Indent `)` the same as the start of the line where `(` is.